CHANGELOG.md

```diff
@@ -62,6 +56,11 @@ output or status of your request. If you want to directly observe the progress
 of the migration as it runs, you can view the CloudWatch logs for your async
 operations (e.g. `PREFIX-AsyncOperationEcsLogs`).
 
+#### CUMULUS-3779 async_operations Docker image version upgrade
+  
+The `async_operations` Docker image has been updated to support Node v20 and `aws-sdk` v3. Users will need to bump
+the version tag of `async_operations` to at least 52 if using the Docker image.
```

Comment:
Is this published to Dockerhub?   We should make direct reference to the project/tag.

Reply:
Updated with a link directly to the project and tag. I also included a section for the `cumulus-ecs-task` image with a direct link to that version.